packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -78,27 +80,61 @@ export const styles = theme => {
         top: 'initial',
         bottom: 0,
         width: 2,
+        height: '100%',
       },
     },
     /* Styles applied to the track element before the thumb. */
     trackBefore: {
       zIndex: 1,
       left: 0,
       transition: trackTransitions,
+      transformOrigin: '0 100%',
+      '&$rtl': {
+        transformOrigin: '0% 100%',
```

Comment:
Is there actually a difference between `0 100%` and `0% 100%`?

Reply:
I was very confused myself when adding that. It seems like JSS or something else rewrites my styles under rtl. What I want to add is `transform-origin: 100% 100%;` to change direction, however `transformOrigin: '100% 100%'` gets rewritten to `transform-origin: 0 100%;`.

So to get the value I want, I have to write the same value as it normally is.

Reply:
I meant why one with and one without the `%`? One doesn't get rewritten by `jss-rtl`?